net/MapViewController.m

```diff
@@ -280,11 +307,64 @@ -(BOOL)mapView:(GMSMapView *)mapView didTapMarker:(GMSMarker *)marker
     return NO;
 }
 
+-(void)liftUpGeolocationButton
```

Comment:
You can have one method instead of liftUpGeolocationButton and pullDownGeolocationButton like
setGeoLocationButtonTop(15.0f)